src/lib/y2partitioner/dialogs/blk_device_resize.rb

```diff
@@ -248,11 +248,15 @@ def growing?
         selected_size > device.size
       end
 
-      # Whether the device is going to be growed more than 1 GiB
+      # Whether the device is going to be growed more than 50 GiB
+      #
+      # @note Threshold to consider a big growing is defined in the old code, but there is
```

Comment:
this sentence missing probably some "not" like "there is not any kind of explanation" otherwise if there is some kind of explanation, I would like to see it also here.

Reply:
Yes, I wanted to say "there is not any ..."